### PR TITLE
Improve the search field styling in the ``sphinx13`` theme

### PR DESCRIPTION
**Pull request by timhoffm**
Current:
grafik

With this PR:
grafik

With this PR and #12590:
grafik

### CODE REVIEW

**Comment by chrisjsewell** [comment]
will wait for #12590 to review

File: doc/_themes/sphinx13/static/sphinx13.css [hunk 1]
```diff
@@ -218,6 +218,20 @@ div.sphinxsidebar #searchbox {
     margin: 12px 0 20px 0;
 }
 
+div.sphinxsidebar #searchbox input[type="text"] {
+    border-radius: var(--admonition-radius) 0 0 var(--admonition-radius);
+}
+
+div.sphinxsidebar #searchbox input[type="text"] {
+    border-radius: var(--admonition-radius) 0 0 var(--admonition-radius);
+}
```
Comment (AA-Turner) [inline doc/_themes/sphinx13/static/sphinx13.css]: I was about to merge this, but these two rules are identical -- is this intentional?

File: doc/_themes/sphinx13/static/sphinx13.css [hunk 2]
```diff
@@ -218,6 +218,20 @@ div.sphinxsidebar #searchbox {
     margin: 12px 0 20px 0;
 }
 
+div.sphinxsidebar #searchbox input[type="text"] {
+    border-radius: var(--admonition-radius) 0 0 var(--admonition-radius);
+}
+
```
Comment (timhoffm) [inline doc/_themes/sphinx13/static/sphinx13.css:224]:
```suggestion
```

File: doc/_themes/sphinx13/static/sphinx13.css (hunk 1 above)
Comment (timhoffm) [reply doc/_themes/sphinx13/static/sphinx13.css]: Sorry, this was not intentional. I've removed the duplicate code.

**Comment by AA-Turner** [comment]
Thanks Tim!

A

**Pull request merged**